frontend/src/components/DeveloperUtilities.vue

```diff
@@ -60,25 +60,21 @@ function download(dataurl: string) {
 })
 export default class DeveloperUtilities extends Vue {
   private editedThermalRefMask: CropBox | null = null;
-  private recordUserActivity = DeviceApi.recordUserActivity;
   private isRecording = false;
 
-  skipWarmup() {
-    this.$root.$children[0].$children[0].$emit("skip-warmup");
+  set recordUserActivity(enable: boolean) {
+    DeviceApi.RecordUserActivity = enable;
+  }
+  get recordUserActivity() {
+    return DeviceApi.RecordUserActivity;
   }
 
-  async onRecordUserActivity() {
-    DeviceApi.recordUserActivity = this.recordUserActivity;
-    window.localStorage.setItem(
-      "recordUserActivity",
-      this.recordUserActivity ? "true" : "false"
-    );
-    if (!this.recordUserActivity) {
-      const { recording } = await DeviceApi.recorderStatus();
-      if (recording) {
-        DeviceApi.stopRecording(false);
-      }
-    }
+  get disableRecordUserActivity() {
+    return DeviceApi.DisableRecordUserActivity;
+  }
+
+  skipWarmup() {
+    this.$root.$children[0].$children[0].$emit("skip-warmup");
```

Comment:
This seems a little bit nasty, relies on the component structure not changing.  Is there a cleaner way to accomplish this?